scripts/mc_add_observation.py

```diff
@@ -0,0 +1,28 @@
+import argparse,pyuvdata,aipy
```

Comment:
Can you add a hashbang line? `#! /usr/bin/env python`

Also copyright and license notices.